.i18nrc.json

```diff
@@ -24,7 +24,8 @@
     "timelion": "src/legacy/core_plugins/timelion",
     "tagCloud": "src/legacy/core_plugins/tagcloud",
     "tsvb": "src/legacy/core_plugins/metrics",
-    "kbnESQuery": "packages/kbn-es-query"
+    "kbnESQuery": "packages/kbn-es-query",
+    "plugins": "src/plugins"
```

Comment:
Do we really want this to be so generic?  If we matched the existing pattern it should probably be something like 
`"inspector": "src/plugins/inspector"`.  Unless there are newer NP recommendations I'm unaware of?  Although it does seem much easier to use this single line than making everyone add new lines per plugin. And the ui strings look nice. So maybe it is the better way to go!

Reply:
I don't know.

@elastic/kibana-stack-services could you please suggest the right pattern?

Reply:
I would think it would be better to be more specific rather than this one line.  Some plugins will not have any localized strings, so the scan of those would be wasted time.  This does have an advantage of reducing maintenance over the long haul, so maybe that's an OK tradeoff.  @bamieh or @azasypkin any thoughts on this?

Reply:
> I would think it would be better to be more specific rather than this one line.

I definitely agree with this, and moreover previously it was required to enforce proper prefix within the plugin (@Bamieh knows better if it's still the case) and help translators to understand where they can find the label in Kibana to test (they receive one big JSON file for the entire Kibana with thousands of labels and having meaningful prefixes should help).

Reply:
Thank you all, made it `"inspector": "src/plugins/inspector"` as suggested.